csharp/ql/src/semmle/code/csharp/ir/implementation/EdgeKind.qll

```diff
@@ -58,6 +71,10 @@ class ExceptionEdge extends EdgeKind, TExceptionEdge {
   final override string toString() { result = "Exception" }
 }
 
+/**
+ * Gets the single instance of `ExceptionEdge`, representing the successor of an instruction when
+ * that instruction's evaluation throws an exception.
```

Comment:
Yeah, sorry for all the duplicate code. It's an artifact of how we share the IR code with C#. We have a script and PR check that keeps them all in sync, so once I make the change in one place, it will be propagated to all the other duplicate copies as well.

Reply:
Sorry - it wasn't intended to be complaint - more an explanation of why I'd stopped commenting 😄